eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -119,6 +119,11 @@ jobs:
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
+        Windows_NetCoreApp_ProjectReferences:
```

Comment:
Why a new target? Shouldn't this be something temporarily added to the .csproj project file?

Reply:
It is for local development but we also want to ensure we have at least one test leg always verifying this scenario.